godtools/App/Services/Renderer/Views/MobileContent/Views/MobileContentViewModel.swift

```diff
@@ -53,27 +53,19 @@ class MobileContentViewModel: NSObject {
         return renderedPageContext.language.direction == .leftToRight ? .left : .right
     }
     
-    func viewDidAppear(analyticsEvents: [MobileContentAnalyticsEvent]) {
+    func viewDidAppear(visibleAnalyticsEvents: [MobileContentAnalyticsEvent]) {
         
-        for event in analyticsEvents {
+        for event in visibleAnalyticsEvents {
             
-            let trigger: AnalyticsEvent.Trigger = event.analyticsEvent.trigger
-
-            if trigger == .visible {
-                event.trigger()
-            }
+            event.trigger()
```

Comment:
Talked with @frett about this last week.  Some events (like user analytics) weren't getting triggered here because `getAnalyticsEvents(type: .visible)` sometimes returns a `.default` type rather than explicitly `.visible` (on pages where visible is the default).  So we just need to make sure to use `getAnalyticsEvents(type)` to do the filtering for us.

Reply:
Thanks @rachaelblue!  So it seems before I was assuming all analytics events (including visible) was getting included hence the check for ```trigger == .visible```.  But, that wasn't the case right?

Reply:
Well, it depended on how you got the analytics events.  In `ToolPageCardViewModel`, you were using `cardModel.analyticsEvents` rather than `cardModel.getAnalyticsEvents(type: .visible)`, so in that case any sort of event could get returned (making the `trigger == .visible` check necessary).  I changed that to `cardModel.getAnalyticsEvents(type: .visible)`, and all the other spots were already using that, so we no longer need the check.

Reply:
Okay that makes sense.  Thank you for updating that.